apollo-config.groovy

```diff
@@ -4,14 +4,17 @@ environments {
     test {
     }
     production {
+//        grails.assets.url = System.getenv("APOLLO_PATH_PREFIX")   +"/assets/"
+//        grails.serverURL = System.getenv("APOLLO_BASE_HOST") + System.getenv("APOLLO_PATH_PREFIX")
+
         dataSource {
             dbCreate = "update"
-            username = "apollo"
-            password = "apollo"
+            username = System.getenv("WEBAPOLLO_DB_USERNAME") ?: "apollo"
+            password = System.getenv("WEBAPOLLO_DB_PASSWORD") ?: "apollo"
 
             driverClassName = "org.postgresql.Driver"
             dialect = "org.hibernate.dialect.PostgresPlusDialect"
-            url = "jdbc:postgresql://127.0.0.1/apollo"
+            url = System.getenv("WEBAPOLLO_DB_URI") ?: "jdbc:postgresql://127.0.0.1/apollo"
```

Comment:
there's no postgres integrated into this image any more, so this default is not a good value. Just take the system env var here?